go.mod

```diff
@@ -227,8 +227,6 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
-replace github.com/lightninglabs/lightning-terminal/autopilotserverrpc => ./autopilotserverrpc
-
```

Comment:
shall we maybe leave the replace for easy dev?

Reply:
ah - nvm, saw chat offline

Reply:
sgtm!